src/contexts/build.context.tsx

```diff
@@ -174,7 +174,7 @@ async function deleteBuild(dispatch: Dispatch, id: string) {
 }
 
 async function stopBuild(dispatch: Dispatch, id: string) {
```

Comment:
would you like to keep it together with `modifyBuild`?

Reply:
Done.